docs/astrocut/index.rst

```diff
@@ -356,10 +356,56 @@ Note that multithreading is disabled by default.
                 Total time: 7.8 sec
 
 
+
+ASDF File Cutouts
+===================
+
+The Nancy Grace Roman Space Telescope will store its data using the Advanced Scientific Data Format (ASDF). With the `asdf_cut` function, users can create cutouts of Roman mission products.
+
+Creating ASDF Cutouts
+----------------------
+
+The function `asdf_cut` performs a cutout of an ASDF file and returns the result as either a FITS file or an ASDF file. The cutout ASDF file format is described `here <file_formats.html#asdf-cutout-files>`__.
+
+.. code-block:: python
+
+                >>> from astrocut import asdf_cut
+                >>> from astropy.coordinates import SkyCoord
+                >>> from astropy.io import fits
+                
+                >>> input_file = ""  # Path to local ASDF file or URI
+
+                >>> center_coord = SkyCoord("80.15189743 29.74561219", unit='deg')
+                
+                >>> cutout_file = asdf_cut(input_file, center_coord.ra, center_coord.dec, cutout_size=200, output_file="roman-demo.fits")  #doctest: +SKIP
```

Comment:
Just curious- what does `#doctest: +SKIP` do in this and other lines?

Reply:
Good question! This tells `doctest` (a module that executes Python code in documentation to verify that it works as intended) to ignore this line. Because I don't specify an actual input file in these calls, they don't execute properly and will throw errors without the `SKIP` directive.